h2/utilities.py

```diff
@@ -41,7 +41,7 @@
 ])
 
 
-def secure_headers(headers):
+def secure_headers(headers, hdr_validation_flags):
```

Comment:
It's not a public function. =)

hyper-h2 is very clear in its documentation: publicness is not signalled by underscores per-se, but by what is documented in the API docs. This is not documented in those docs (in fact, the entire `utilities` module is not). Therefore, we can do whatever we want here.